csharp/test/Ice/optional/AllTests.cs

```diff
@@ -108,11 +107,11 @@ internal static ITestPrx Run(TestHelper helper)
             test.OpBasicIn(17, 17, null, "test");
             test.OpBasicIn(17, null, null, "test");
 
-            (int? r, int o1, int? o2, string? o3) = test.OpBasicInOut(5, 15, "test");
-            TestHelper.Assert(r!.Value == 15 && o1 == 5 && o2!.Value == 15 && o3! == "test");
+            // (int? r, int o1, int? o2, string? o3) = test.OpBasicInOut(5, 15, "test");
```

Comment:
Why is this commented out? Since you renamed the operation, the generated method(s) are now called `OpBasicInReturnTuple`.

Reply:
Yes, you should uncomment them and change the name from `OpBasicInOut` to `OpBasicInReturnTuple`